server/src/main/java/org/apache/iotdb/db/engine/compaction/CompactionTaskManager.java

```diff
@@ -238,7 +239,8 @@ public synchronized void submitTaskFromTaskQueue() {
     try {
       while (currentTaskNum.get()
               < IoTDBDescriptor.getInstance().getConfig().getConcurrentCompactionThread()
-          && !candidateCompactionTaskQueue.isEmpty()) {
+          && !candidateCompactionTaskQueue.isEmpty()
+          && candidateCompactionTaskQueue.toString() != null) {
```

Comment:
Why use `candidateCompactionTaskQueue.toString()` as a condition here?

Reply:
This is a debug statement so that I can get  the return value of this statement through arthas. I will remove this statement later.

Reply:
Got it.